src/app/components/Context/SplittingMethod.tsx

```diff
@@ -20,16 +20,21 @@ const options: {
 ];
 
 const SplittingMethod = () => {
-  const { setValue } = useFormContext();
+  const { setValue, register, watch } = useFormContext();
+
+  // Initialize the value if needed
+  const currentMethod = watch('splittingMethod');
+
+  // Register the field so react-hook-form knows about it
+  register('splittingMethod');
```

Comment:
will add controller later on to streamline the registration

https://react-hook-form.com/docs/usecontroller/controller


FYI, not smth to act on now